src/main/java/seedu/address/model/student/academic/Attendance.java

```diff
@@ -12,30 +12,30 @@
     public static final String DATE_CONSTRAINTS =
             "Attendance dates should be valid and in the form dd/mm/yy, and should not be blank";
     public static final String STATUS_CONSTRAINTS =
-            "Attendance status should be either 'attended' or 'unattended'.";
+            "Attendance status should be either 'present' or 'absent'.";
     private static final DateTimeFormatter INPUT_DEF = DateTimeFormatter.ofPattern("d/M/yy");
     private static final DateTimeFormatter INPUT_ALT = DateTimeFormatter.ofPattern("d/M/yyyy");
     private static final DateTimeFormatter OUTPUT = DateTimeFormatter.ofPattern("dd MMM yyyy");
-    private static final String ATTENDED_STATUS = "attended";
-    private static final String UNATTENDED_STATUS = "unattended";
+    private static final String PRESENT_STATUS = "present";
+    private static final String ABSENT_STATUS = "absent";
```

Comment:
What do you say to using U2713 (✓) and U+2718 (✘) for attendance? something like Question where it will just be `(tick/cross) DATE` to show attendance in a single line

That or perhaps rename "Status" to "Attendance" instead, it might be more intuitive than the former.

Reply:
I think i'll use the tick and cross to match up with yours, but i think feedback will have to be on a separate line (for now)
Also I feel that I don't want to repeat the word "Attendance" multiple times, cos i use it to name the section already.

Reply:
Changed it to this: 
![image](https://user-images.githubusercontent.com/60608393/97666589-fc0dce80-1ab8-11eb-970f-8c64f40a789f.png)